### PR TITLE
Identifiers - edits

### PR DESCRIPTION
**Pull request by brentzundel**
Edited for clarity, line length, spacing, and to add links to terminology.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -144,28 +144,34 @@ <h2>Introduction</h2>
...
-          When expressing statements about a specific entity, such as a person, product, or organization,
-          it is often useful to use some kind of identifier so that others can express statements about
-          the same thing. You may have noticed that the VC Data Model specification contains numerous
-          examples where the choice of identifier for the subject of the credential (the thing the claims
-          are about) has been a <a href="https://w3c-ccg.github.io/did-primer/">decentralized identifier</a>,
-          also known as a DID. An example of a DID is <code>did:example:123456abcdef</code>.
+When expressing statements about a specific entity, such as a person, product,
+or organization, it is often useful to have an identifier for it so that others
+can express statements about the same thing. The <a>verifiable credentials</a>
+<a href="https://w3c.github.io/vc-data-model/">data model specification</a>
```
Comment (TallTed) [inline index.html]: `w3c.github.io`-based links should be changed to `w3.org`-based links.

File: index.html [hunk 2]
```diff
@@ -144,28 +144,34 @@ <h2>Introduction</h2>
...
+As of the publication of the <a>verifiable credentials</a>
+<a href="https://w3c.github.io/vc-data-model/">data model specification</a>,
+DIDs are not necessary for <a>verifiable credentials</a> to be useful.
+Specifically, <a>verifiable credentials</a> do not depend on <a>DIDs</a> and
+<a>DIDs</a> do not depend on <a>verifiable credentials</a>. However, it is
+expected that many <a>verifiable credentials</a> will use <a>DIDs</a> and that
+software libraries implementing the
+<a href="https://w3c.github.io/vc-data-model/">data model specification</a>
+will benefit from knowing how to resolve <a>DIDs</a>. <a>DID</a>-based URLs are
+used for expressing identifiers associated with <a>subjects</a>, <a>issuers</a>,
```
Comment (TallTed) [inline index.html]:
```
URLs are
used for expressing
```
->
```
URLs may 
be used to express
```

Idea is to not suggest that URLs expressing these identifiers should (only) be DID-based.

Comment (brentzundel) [reply index.html]: Good catch, thank you. Fixed in [b1c845b](https://github.com/w3c/vc-imp-guide/pull/27/commits/b1c845b435ab6438c1e6e2eed60e213d4aa2f037)

File: index.html (hunk 1 above)
Comment (brentzundel) [reply index.html]: Changed in [b1c845b](https://github.com/w3c/vc-imp-guide/pull/27/commits/b1c845b435ab6438c1e6e2eed60e213d4aa2f037)

**Review by ken-ebert — APPROVED** [review]

**Pull request merged**